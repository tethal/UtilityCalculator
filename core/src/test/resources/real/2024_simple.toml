[general]
start_date = 2024-01-01
end_date = 2024-12-31
tenant = ["Marie Černá", "Jindřišská 16", "111 50 Praha 1"]
owner = ["Jan Novák", "majitel@example.com"]
report_place = "V Praze"
report_date = 2025-05-08
sources = ["Vyúčtování služeb od Společenství vlastníků za rok 2024"]

[[deposits]]
description="leden - prosinec"
count=12
amount=3000

[[other_fees]]
start_date = 2024-01-01
end_date = 2024-12-31
annual_cost = 39514.3
