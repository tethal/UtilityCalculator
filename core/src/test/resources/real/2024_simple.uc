2024-01-01...2024-31-12
V Praze, 2025-05-08

[najemnik]
    Marie Černá
    Jindřišská 16
    111 50 Praha 1

[najemce]
    Jan Novák
    majitel@example.com

[zdroje]
    Vyúčtování služeb od Společenství vlastníků za rok 2024


[zalohy]
    leden - prosinec: 12 x 3000

[ostatni poplatky:Náklady]
    2024: 2344.98+360+510.97+27.84+44.6+2502.39+1182.07+1413.39+1043.27+653.08+12431.84+8195.27+2604.44+5920.16+280
