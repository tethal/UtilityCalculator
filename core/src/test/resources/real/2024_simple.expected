Report:
  dateRange: 2024-01-01...2025-01-01
  tenant: Marie Černá, Jindřišská 16, 111 50 Praha 1
  owner: Jan Novák, majitel@example.com
  placeDate: V Praze, 2025-05-08
  sources: Vyúčtování služeb od Společenství vlastníků za rok 2024
  sections:
    Ostatní poplatky: 39514.30
    Přijaté zálohy: -36000
    TOTAL: 3514.30

Ostatní poplatky:
  2024-01-01...2025-01-01: 12.00 * 3292.86 = 39514.30

Přijaté zálohy:
  leden - prosinec: 12 * 3000 = 36000
